cdk/lib/support-workers.ts

```diff
@@ -193,6 +193,9 @@ export class SupportWorkers extends GuStack {
           parameterStorePolicy,
           ...additionalPolicies,
         ],
+        environment: {
+          AWS_LAMBDA_LOG_FORMAT: "Unstructured",
```

Comment:
The logging format in lambdas has recently changed to JSON. This puts it back to text logging which I think is easier to read.